WordPress/src/main/java/org/wordpress/android/util/image/ImageManager.kt

```diff
@@ -148,6 +151,27 @@ class ImageManager @Inject constructor(private val placeholderManager: ImagePlac
                 .clearOnDetach()
     }
 
+    /**
+     * Loads an image from the "imgUrl" into the ImageView and applies circle transformation. Adds placeholder and
+     * error placeholder depending on the ImageType.
+     */
+    fun loadWithRoundedCorners(
+        imageView: ImageView,
+        imageType: ImageType,
+        imgUrl: String,
+        roundingRadius: Pixels
+    ) {
+        val context = imageView.context
+        if (!context.isAvailable()) return
+        GlideApp.with(context)
+                .load(imgUrl)
+                .addFallback(imageType)
+                .addPlaceholder(imageType)
+                .transform(RoundedCorners(roundingRadius))
```

Comment:
Something I noticed while doing this implementation, is that the `rounded corners` transformation is not applied to placeholders. I I checked the `Glide` github repo and they say it's not supported. Should I utilize an XML approach to perform the rounding required based on the designs? 

I just realized that elsewhere in the app in particular `MySiteFragment`, a material card view is being utilized to accomplish this so I might opt for that approach instead. 

```xml
<com.google.android.material.card.MaterialCardView
                        android:layout_width="@dimen/my_site_blavatar_sz"
                        android:layout_height="@dimen/my_site_blavatar_sz"
                        android:layout_margin="@dimen/margin_small"
                        app:cardCornerRadius="@dimen/my_site_blavatar_corner_radius"
                        app:cardElevation="0dp">
```

Let me know 😉

Reply:
I'd consider one of the following approaches
1. Utilize the MaterialCardView - I personally don't see any downside to it and it seems like the simplest solution.
2. Add a new optional sealed class paramater to `fun load(imageView: ImageView, imageType: ImageType, imgUrl: String = "", scaleType: ScaleType = CENTER)`.  Similarly to `ScaleType`. Unfortunately, as you mentioned this won't work for placeholders :(.

Wdyt?

Reply:
Thanks! I think the `CardView` is the simplest approach so I opted for it.
51ac8d9

Reply:
👍I believe we can remove `loadWithRoundedCorners` from the ImageManager now, right? Wdyt?

Reply:
yes, I thought I had removed that 😅